### PR TITLE
[MISC] Clean template

### PR DESCRIPTION
**Pull request by NicolaDonelli**
Closes issue #16 refactoring also the README and some project files that should have been made parametric

### CODE REVIEW

**Comment by deusebio** [comment]
Looks good! While playing around with the branch I've noticed a minor bug that maybe you could correct within this PR. 

[here](https://github.com/CGnal/python-ci-template/blob/c54b296e62efc390542617fad2b287e98a949103/templates/setup.cfg.tmpl#L10) you see that the coverage is measured against the test folder which does not make much sense. It should be against the src folder. We should put a placeholder here too in the template

Otherwise, good to be merged!

**Review by deusebio — APPROVED** [review]

File: Dockerfile [hunk 1]
```diff
@@ -0,0 +1,14 @@
...
+WORKDIR /app
+
+COPY requirements/requirements.txt requirements.txt
+RUN pip3 install -r requirements.txt
+
+COPY . .
+
+WORKDIR /app/src
+
+CMD [ "python3", "-m" , "flask", "run", "--host=0.0.0.0"]
```
Comment (deusebio) [inline Dockerfile:14]: Ah, this does not make much sense anymore since there is no rest api running

File: setup.cfg [hunk 2]
```diff
@@ -4,18 +4,27 @@ style = pep440
 versionfile_source = src/_version.py
 versionfile_build = src/_version.py
 tag_prefix =
+parentdir_prefix = python-ci-template
 
 [tool:pytest]
 addopts = --doctest-modules --cov=./tests
```
Comment (deusebio) [inline setup.cfg:10]: should be --cov=./src

**Comment by deusebio** [comment]
Just fix those minor points and then you can go ahead directly with merging the PR

File: setup.cfg (hunk 2 above)
Comment (NicolaDonelli) [reply setup.cfg:10]: Yup you are right... dunno why it had always been like that... I'll fix it!

File: Dockerfile (hunk 1 above)
Comment (NicolaDonelli) [reply Dockerfile:14]: You are right. I'll delete it

**Comment by aiola** [comment]
Looks good! Only a minor comment: since we have setup.cfg.tmpl, why do we keep also setup.cfg?

**Review by aiola — APPROVED** [review]

**Pull request merged**